CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [Rolling Pool Update] Don't fail if `load-balancer` plugin is missing (Starter and Enterprise plans)
+- [web] Show more significant numbers when relevant (for example 1.28 TB of memory instead of 1TB)
```

Comment:
The `[web]` scope is not relevant, as it encompasses almost everything in the changelog (what the users sees), either find something more specific if relevant or remove it :slightly_smiling_face: